Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -42,16 +42,25 @@
     color: var(--text-color-secondary);
 }
 
-.details-section > .content > .group > .row.font-variation > .value {
+.details-section > .content > .group > .row.font-variation > input[type="number"] {
     grid-column: axis-value;
-    text-align: end;
+    height: max-content;
+    width: 6ch;
```

Comment:
This was a misguided attempt at ensuring that at least 6 characters from the value are visible without being cut off.
That doesn't work because the font is not monospace. Will revert to using pixels.